src/connectors/aws_inventory.py

```diff
@@ -192,22 +192,22 @@ def ingest(table_name, options):
     source_role_arn = options.get('source_role_arn')
     destination_role_name = options.get('destination_role_name')
     external_id = options.get('external_id')
-    accounts_identifier = options.get('accounts_identifier')
+    accounts_connection_name = options.get('accounts_connection_name')
```

Comment:
After this line, can you add a quick check to see if the option starts with `data.`, and prepend `data.` if it doesn't? That way, it'll support the user either providing or omitting the schema, and will help prevent bug reports down the line.